build.zig

```diff
@@ -51,7 +64,7 @@ pub fn build(b: *std.build.Builder) !void {
     exe.install();
```

Comment:
Lastest zig (0.11.0-dev) update, replacing:
* `exe.install()` to `b.installArtifact(exe)` to build
* `exe.run()` to `b.addRunArtifact(exe)` to run